javascript/modules/client/src/client.ts

```diff
@@ -124,6 +124,7 @@ export class Client extends ClientCore implements IClient {
   estimation = {
     create: (params: ICreateParams) => this._estimateCreation(params),
     deposit: (params: IDepositParams) => this._estimateDeposit(params),
+    increaseAllowance: (params: IDepositParams) => this._estimateIncreaseAllowance(params),
```

Comment:
Why are we adding this? Is this to estimate the `approve` operation within `deposit()` ?
In such case, let's add an IF clause within the `deposit()` function and estimate the increased approval if not enough money is approved.